article/Rougier-2017.md

```diff
@@ -0,0 +1,132 @@
+in computational biology or neuroscience where the distribution of a cell
+population can be conveniently described by shades of grey. It is to be noted
+that since the publication of this paper, a number of related techniques have
+been
+proposed [@AscencioLopez:2010][@Balzer:2009a][@Balzer:2009b][@Bridson:2007]
+that may be more efficient but with lower quality. We thus privileged this
+implementation that is simple and provides high stippling quality.
+
+# Methods
+
+We applied the method proposed in the original paper with some variations due
+to the non-use of the GPU for computing the Voronoi diagram. In the original
+article, author computes the Voronoi diagram by taking advantage of the graphic
+card (GPU) and the fact that a set of cones seen from above actually represents
+a Voronoi diagram (see [@Hoff:1999]). We use instead
+the [QHull](http://www.qhull.org) library (through
+the [Scipy](https://scipy.github.io) Python package) for computing the Voronoi
+diagram together with the Voronoi regions (as a list of segments). We also took
+care of adding extra points such that each cell is contained within a
+user-defined bounding box that will be set to the dimension of the density
+image. For computing the weighted centroid, we applied the definition proposed
+in the original paper over the discrete representation of the domain:
+
+$${\bf C}_i = \frac{\int_A {\bf x}\rho({\bf x})dA}{\int_A \rho({\bf x})}$$
+
+However, we did not use the proposed optimization. Instead, each cell is
+rasterized (as a set of pixels) and the centroid is computed using the
+integrals over the whole set of pixels composing the Voronoi cell. As noted by
+the author, the precision of the method is directly related to the size of the
+Voronoi cell. Consequently, if the original density image is too small
+relatively to the number of stipples, there might be quality issues. Finally,
+we used a fixed number of iteration ($n=50$) instead of using the difference in
+the standard deviation of the area of the Voronoi regions as in the original
+paper since the definition of the rejection criterion was not clear in the
+original article and quite arbitrary.
+
+# Results
+
+We contacted the original author asking for permission to re-use
+the
+[original images](http://cs.nyu.edu/~ajsecord/npar2002/StipplingOriginals.zip)
+but did not obtain any response. We thus display here only the output of our
+replication to be compared with the original ones. The climbing shoe (figure
+@fig:shoe) and the corn plant (figure @fig:corn) are very similar to the images
+displayed in the original article. However, for the large and small Peperomia
+plants (figure @fig:large-plant & @fig:small-plant), the output of our
+replication is clearly at a lower quality without having identified the
+cause. Most probably, the limited resolution of the input image may be a
+critical factor and it is not clear if the author used these small resolution
```

Comment:
How likely is it that the cause is related to the differences in the method, e.g. the omission of the optimization mentioned above? (Or was that a purely performance optimization?)